src/cgame/cg_buildable.cpp

```diff
@@ -2229,11 +2229,12 @@ void CG_Buildable( centity_t *cent )
 
 	if ( !( es->eFlags & EF_B_SPAWNED ) )
 	{
-		sfxHandle_t prebuildSound = cgs.media.humanBuildablePrebuild;
+		sfxHandle_t prebuildSound;
 
 		switch ( team )
 		{
 		case TEAM_HUMANS:
+		default:
```

Comment:
maybe:

```
default: // make the compiler happy by making it knows prebuildSound is always set
```

or something like that (if that was the issue)?

Reply:
I've dropped the commit for now, I guess it will need proper multiple team handling anyway.